lib/Doctrine/DBAL/Platforms/PostgreSqlPlatform.php

```diff
@@ -752,6 +752,11 @@ public function getDropForeignKeySQL($foreignKey, $table)
         return $this->getDropConstraintSQL($foreignKey, $table);
     }
 
+    protected function getDropPrimaryKeySQL(string $table) : string
+    {
+        return $this->getDropConstraintSQL("{$table}_pkey", $table);
```

Comment:
Please use string concatenation here + `'` instead of variable in string.

Reply:
I found a [couple](http://razorsql.com/features/postgresql_drop_primary_key.html) of [articles](https://gist.github.com/scaryguy/6269293) explaining that this is the way how the PK should be dropped on PostgreSQL but before seeing the explanation, this approach looks like using a "magic" suffix.

It would be nice if a link to documentation describing this approach was provided in the doc-block.